web/react/components/edit_channel_header_modal.jsx

```diff
@@ -11,12 +12,15 @@ export default class EditChannelHeaderModal extends React.Component {
     constructor(props) {
         super(props);
 
-        this.handleEdit = this.handleEdit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
 
         this.onShow = this.onShow.bind(this);
         this.onHide = this.onHide.bind(this);
 
         this.state = {
+            originalHeader: props.channel.header,
+            header: props.channel.header,
```

Comment:
`this.state.header` is the value of the textbox on the page. I switched the textarea to be controlled (ie it has a backing state value) back when I thought the cause of the problem was because we were only changing the defaultValue. That didn't turn out to be the cause of the problem, but I left it in since controlled inputs seem more idiomatic